tests/utils/plaintexttohtml.js

```diff
@@ -10,21 +10,12 @@ describe( 'plainTextToHtml()', () => {
 		expect( plainTextToHtml( 'x y <z>' ) ).to.equal( 'x y &lt;z&gt;' );
 	} );
 
-	it( 'turns double line breaks into paragraphs', () => {
```

Comment:
It's usually good to leave all the tests of the previous implementation. Right now, we know that a single line break turns into a paragraph. But what about two? This is still a pretty distinctive case. Also, since prior to your changes  there was a code which handled double line breaks, how do you know now that it's completely removed? There's actually a non-theoretical chance that that code remained there.

Reply:
@Reinmar case with 1, 2, 3 new lines is tested in next test "turns combination of different amount of line breaks to paragraphs":
`a\nb\n\nc\n\n\nd`

Reply:
The purpose of the last test is to see if a varying number of line breaks doesn't e.g. cause some tags to be left open.